interface/globals.php

```diff
@@ -198,7 +198,7 @@ function undoMagicQuotes($array, $topLevel=true) {
 $GLOBALS['current_version_js_path'] = "$web_root/assets/js/current_version";
 
 //module configurations
-$GLOBALS['modules_dir']  = "$webserver_root/modules/";  //CURRENT modules directory.
+$GLOBALS['modules_dir']  = "$webroot/modules/";        //CURRENT modules directory.
```

Comment:
Perfect! ... sometimes we miss little things :). I am approving this works fine and LGTM!

Reply:
You have trailing whitespace, could you clean that up, please?

Reply:
Ingore previous comment